smartsim/_core/entrypoints/service.py

```diff
@@ -40,21 +40,34 @@ class Service(ABC):
     hooks for status changes"""
 
     def __init__(
-        self, as_service: bool = False, cooldown: int = 0, loop_delay: int = 0
+        self,
+        as_service: bool = False,
+        cooldown: int = 0,
+        loop_delay: int = 0,
+        health_check_frequency: float = 0,
     ) -> None:
         """Initialize the ServiceHost
+
         :param as_service: Determines if the host will run until shutdown criteria
         are met or as a run-once instance
         :param cooldown: Period of time to allow service to run before automatic
```

Comment:
```
        :param cooldown: Period of time (in seconds) to allow the service to run
         after a shutdown is permitted. Enables the service to avoid restarting if
         new work is discovered. A value of 0 disables the cooldown.
```